setup.py

```diff
@@ -43,17 +43,21 @@
 with open('requirements.txt') as f:
     required = f.read().splitlines()
 
-setup(name='basin3d-core',
+setup(name='basin3d',
       version=__release__,
       description='BASIN-3D Core Framework',
       long_description=README,
-      author='Val Hendrix',
-      author_email='vchendrix@lbl.gov',
-      url='https://github.com/Watershed-Function-SFA/basin3d-core',
+      author='Val Hendrix, <list all authors>',
+      author_email='vchendrix@lbl.gov, <list all author emails>',
```

Comment:
Add the actual authors and their emails here.

Reply:
I will make those changes and look for where that 'basin3d-core 0.0.1' could be

Reply:
@dschristianson which page(s) did you open where you see basin3d-core 0.0.1?

Reply:
![Screen Shot 2021-04-02 at 12 06 36 PM](https://user-images.githubusercontent.com/4153416/113446237-423ec500-93ac-11eb-8a23-56fa7471e17a.png)

takes me here

![Screen Shot 2021-04-02 at 12 07 54 PM](https://user-images.githubusercontent.com/4153416/113446250-4965d300-93ac-11eb-8409-da6f87c3f557.png)